packages/server/modules/core/tests/users.spec.js

```diff
@@ -151,18 +152,20 @@ describe('Actors & Tokens @user-services', () => {
     it('Find or create should create a user', async () => {
       const newUser = {}
       newUser.name = 'Steve Ballmer Balls'
-      newUser.email = 'ballmer@balls.com'
+      newUser.email = 'ballmer@example.test'
       newUser.password = 'testthebest'
 
       const { id } = await findOrCreateUser({ user: newUser })
       ballmerUserId = id
       expect(id).to.be.a('string')
+      const user = await getUserById({ userId: id })
+      expect(user.verified).to.equal(true)
     })
 
     it('Find or create should NOT create a user', async () => {
       const newUser = {}
       newUser.name = 'Steve Ballmer Balls'
-      newUser.email = 'ballmer@balls.com'
+      newUser.email = 'ballmer@example.test'
```

Comment:
I needed to change this because we have 2 tests depending on each other because of the variable `ballmerUserId`.

Reply:
I've used vitest in the past too, not sure how to set that option in mocha.
Anyway for what I can see, there are 2 tests here depending on each other so running those tests in a different order will make them fail.